### ISSUE TITLE
Can't get the password when having multiple items for one site

### ISSUE THREAD

**Issue by rasmusbe** [blacs30/bitwarden-alfred-workflow#3]
When selecting an item where there are multiple items for the same site I don't get the password in my clipboard – I get an error.

I'm using bitwarden-cli 1.3.0 and bitwarden-alfred-workflow 1.2.2.

# Steps to reproduce
1. **Select an item where there are multiple logins for one site and press enter. I have blurred my usernames but they are all different.**
![image](https://user-images.githubusercontent.com/2124386/46143304-d5d23080-c259-11e8-9987-745d73f1ea2c.png)

2. **This is copied to my clipboard**
```
More than one result was found. Try getting a specific object by `id` instead. The following objects were found:
56d6067b-35de-4703-a766-a942008409a4
48c3a343-fdec-400d-a995-a942008409a4
e7c0f431-374a-47b0-886f-a942008409a5
cfab1cdc-c2f7-4efb-91b1-a95900dd06cb
98cf272d-5c08-4d14-bd44-a942008409a4
800b6ccc-8bc9-4739-941f-a942008409a5
c1f19717-f58d-4fcc-8ba2-a95f00bd191f
71a6d990-c85f-46d8-a85f-a942008409a4
89f26395-d54e-4fd7-a6e7-a942008409a4
013b666a-46e2-4746-8163-a942008409a5
800fe1cc-85d0-426d-9e34-a942008409a5
da57cb75-20c5-4597-80ca-a942008409a5
3dac775c-8242-43a4-8412-a942008409a5
a6351a17-c71c-490e-89f2-a942008409a5
1225e4da-c29c-4479-be78-a942008409a4
5e389ccb-19f8-4592-9898-a942008409a5
667bab13-8b68-4ffc-b870-a942008409a4
f12a1f07-b5cd-477a-9741-a942008409a5
0b89a0e9-d757-467d-8f6b-a942008409a5
7c528581-325e-4a41-9246-a942008409a5
0dc96acf-d6a4-407e-b386-a942008409a5
cf252dbf-7a5a-4d01-bffa-a942008409a5
60909c7e-23b5-4ef8-9570-a942008409a5
3b4d3fb7-d950-4e9d-b079-a942008409a5
1166afe4-e0bd-4cd3-b4f1-a942008409a5
e1ea5d23-8e6d-4bab-a06f-a942008409a5
ef7bfe90-ce6c-45f2-92ee-a942008409a5
fbe82418-930d-484d-a6ae-a942008409a5
037f7dc6-00fd-4514-b0ad-a942008409a5
e790dc82-49e1-41f5-b75f-a942008409a4
fc09ad54-ee21-4fe9-973f-a942008409a5
a6b1c6ba-60f8-46ea-b806-a942008409a4
f87a8891-58d7-4d88-a15e-a942008409a4
7496233f-97ec-42f3-a599-a942008409a5
215352eb-f74d-4c5a-aa5f-a942008409a5
357e7ec7-91f1-4db8-9b75-a942008409a5
1c6043b7-42ce-4a25-9708-a942008409a5
9a7f6786-b326-47a3-b569-a942008409a5
de0daddb-c002-4b04-ba56-a942008409a5
80d6cc4a-2231-4a59-8b1f-a942008409a5
a007c866-0d98-4996-bfe7-a942008409a5
72ff4287-9108-4072-81fd-a942008409a4
f6130bda-9fe1-4135-bcdf-a942008409a5
db9890a7-a4bb-4936-b5de-a942008409a5
42d9ae52-7fbf-4d4c-aa0d-a942008409a4
5815f5a6-9b06-4f88-b4ab-a942008409a5
998de16d-a5c4-4dfc-af31-a942008409a4
c53013d8-f699-4538-8978-a942008409a5
8528bbd9-c589-4171-8356-a942008409a5
7685481d-86d7-43d2-b38f-a942008409a5
afe27519-563e-47c7-a0c7-a942008409a5
befe90d5-30ca-459e-9dac-a942008409a5
df4dfcd7-48c1-4b1d-b543-a942008409a4
741ae853-c19d-4189-8119-a942008409a4
8b77b39d-3005-483e-b2e8-a942008409a5
f2641646-c12d-4d57-9e97-a942008409a5
182e7949-9fea-49fb-9b8a-a942008409a5
3e408f99-2143-4ce3-9a71-a942008409a4
819254c8-35a4-45d8-a9da-a942008409a5
b89a3130-b5d7-4b95-b0a9-a942008409a5
74e88442-aa45-4762-8da0-a942008409a5
89956967-45bc-4288-8286-a942008409a5
ed46284e-8c7d-4f36-9d5f-a942008409a5
6df44ae0-11fa-4695-873e-a942008409a5
aceae15c-4219-42af-ad16-a942008409a5
b77902ba-0837-49d2-8bea-a942008409a5
199568ea-6243-4f67-b9bd-a942008409a5
8433d059-9637-4f9b-bdb7-a96700c9deec

```

**Comment by rasmusbe** [comment]
<details>
<summary><b>I found the debug log for this search:</b></summary>
```
Starting debug for 'Bitwarden CLI'

[2018-09-27 13:48:55][input.scriptfilter] Queuing argument 'apple'
[2018-09-27 13:48:56][input.scriptfilter] Script with argument '(null)' finished
[2018-09-27 13:48:56][input.scriptfilter] {"items": [{"arg": ["apple.com", "71a6d990-c85f-46d8-a85f-a942008409a4"], "subtitle": "Username: account1@domain.com. OTP: ---", "uid": "71a6d990-c85f-46d8-a85f-a942008409a4", "icon": {"path": "icon.png"}, "title": "apple.com"}, {"arg": ["apple.com - account3@domain.com", "89f26395-d54e-4fd7-a6e7-a942008409a4"], "subtitle": "Username: account3@domain.com. OTP: ---", "uid": "89f26395-d54e-4fd7-a6e7-a942008409a4", "icon": {"path": "icon.png"}, "title": "apple.com - account3@domain.com"}, {"arg": ["apple.com - account2@domain.com", "013b666a-46e2-4746-8163-a942008409a5"], "subtitle": "Username: account2@domain.com. OTP: ---", "uid": "013b666a-46e2-4746-8163-a942008409a5", "icon": {"path": "icon.png"}, "title": "apple.com - account2@domain.com"}, {"arg": ["appleid.apple.com", "94eff414-ed5c-4af2-bac4-a942008409a5"], "subtitle": "Username: account4@domain.com. OTP: ---", "uid": "94eff414-ed5c-4af2-bac4-a942008409a5", "icon": {"path": "icon.png"}, "title": "appleid.apple.com"}]}
[2018-09-27 13:49:00][input.scriptfilter] Processing output of 'action.script' with arg '(
    "apple.com - account3@domain.com",
    "89f26395-d54e-4fd7-a6e7-a942008409a4"
)'
[2018-09-27 13:49:01][action.script] Processing output of 'output.clipboard' with arg 'More than one result was found. Try getting a specific object by `id` instead. The following objects were found:
56d6067b-35de-4703-a766-a942008409a4
48c3a343-fdec-400d-a995-a942008409a4
e7c0f431-374a-47b0-886f-a942008409a5
cfab1cdc-c2f7-4efb-91b1-a95900dd06cb
98cf272d-5c08-4d14-bd44-a942008409a4
800b6ccc-8bc9-4739-941f-a942008409a5
c1f19717-f58d-4fcc-8ba2-a95f00bd191f
71a6d990-c85f-46d8-a85f-a942008409a4
89f26395-d54e-4fd7-a6e7-a942008409a4
013b666a-46e2-4746-8163-a942008409a5
800fe1cc-85d0-426d-9e34-a942008409a5
da57cb75-20c5-4597-80ca-a942008409a5
3dac775c-8242-43a4-8412-a942008409a5
a6351a17-c71c-490e-89f2-a942008409a5
1225e4da-c29c-4479-be78-a942008409a4
5e389ccb-19f8-4592-9898-a942008409a5
667bab13-8b68-4ffc-b870-a942008409a4
f12a1f07-b5cd-477a-9741-a942008409a5
0b89a0e9-d757-467d-8f6b-a942008409a5
7c528581-325e-4a41-9246-a942008409a5
0dc96acf-d6a4-407e-b386-a942008409a5
cf252dbf-7a5a-4d01-bffa-a942008409a5
60909c7e-23b5-4ef8-9570-a942008409a5
3b4d3fb7-d950-4e9d-b079-a942008409a5
1166afe4-e0bd-4cd3-b4f1-a942008409a5
e1ea5d23-8e6d-4bab-a06f-a942008409a5
ef7bfe90-ce6c-45f2-92ee-a942008409a5
fbe82418-930d-484d-a6ae-a942008409a5
037f7dc6-00fd-4514-b0ad-a942008409a5
e790dc82-49e1-41f5-b75f-a942008409a4
fc09ad54-ee21-4fe9-973f-a942008409a5
a6b1c6ba-60f8-46ea-b806-a942008409a4
f87a8891-58d7-4d88-a15e-a942008409a4
7496233f-97ec-42f3-a599-a942008409a5
215352eb-f74d-4c5a-aa5f-a942008409a5
357e7ec7-91f1-4db8-9b75-a942008409a5
1c6043b7-42ce-4a25-9708-a942008409a5
9a7f6786-b326-47a3-b569-a942008409a5
de0daddb-c002-4b04-ba56-a942008409a5
80d6cc4a-2231-4a59-8b1f-a942008409a5
a007c866-0d98-4996-bfe7-a942008409a5
72ff4287-9108-4072-81fd-a942008409a4
f6130bda-9fe1-4135-bcdf-a942008409a5
db9890a7-a4bb-4936-b5de-a942008409a5
42d9ae52-7fbf-4d4c-aa0d-a942008409a4
5815f5a6-9b06-4f88-b4ab-a942008409a5
998de16d-a5c4-4dfc-af31-a942008409a4
c53013d8-f699-4538-8978-a942008409a5
8528bbd9-c589-4171-8356-a942008409a5
7685481d-86d7-43d2-b38f-a942008409a5
afe27519-563e-47c7-a0c7-a942008409a5
befe90d5-30ca-459e-9dac-a942008409a5
df4dfcd7-48c1-4b1d-b543-a942008409a4
741ae853-c19d-4189-8119-a942008409a4
8b77b39d-3005-483e-b2e8-a942008409a5
f2641646-c12d-4d57-9e97-a942008409a5
182e7949-9fea-49fb-9b8a-a942008409a5
3e408f99-2143-4ce3-9a71-a942008409a4
819254c8-35a4-45d8-a9da-a942008409a5
b89a3130-b5d7-4b95-b0a9-a942008409a5
74e88442-aa45-4762-8da0-a942008409a5
89956967-45bc-4288-8286-a942008409a5
ed46284e-8c7d-4f36-9d5f-a942008409a5
6df44ae0-11fa-4695-873e-a942008409a5
aceae15c-4219-42af-ad16-a942008409a5
b77902ba-0837-49d2-8bea-a942008409a5
199568ea-6243-4f67-b9bd-a942008409a5
8433d059-9637-4f9b-bdb7-a96700c9deec
'
[2018-09-27 13:49:01][output.clipboard] Processing output of 'output.notification' with arg 'More than one result was found. Try getting a specific object by `id` instead. The following objects were found:
56d6067b-35de-4703-a766-a942008409a4
48c3a343-fdec-400d-a995-a942008409a4
e7c0f431-374a-47b0-886f-a942008409a5
cfab1cdc-c2f7-4efb-91b1-a95900dd06cb
98cf272d-5c08-4d14-bd44-a942008409a4
800b6ccc-8bc9-4739-941f-a942008409a5
c1f19717-f58d-4fcc-8ba2-a95f00bd191f
71a6d990-c85f-46d8-a85f-a942008409a4
89f26395-d54e-4fd7-a6e7-a942008409a4
013b666a-46e2-4746-8163-a942008409a5
800fe1cc-85d0-426d-9e34-a942008409a5
da57cb75-20c5-4597-80ca-a942008409a5
3dac775c-8242-43a4-8412-a942008409a5
a6351a17-c71c-490e-89f2-a942008409a5
1225e4da-c29c-4479-be78-a942008409a4
5e389ccb-19f8-4592-9898-a942008409a5
667bab13-8b68-4ffc-b870-a942008409a4
f12a1f07-b5cd-477a-9741-a942008409a5
0b89a0e9-d757-467d-8f6b-a942008409a5
7c528581-325e-4a41-9246-a942008409a5
0dc96acf-d6a4-407e-b386-a942008409a5
cf252dbf-7a5a-4d01-bffa-a942008409a5
60909c7e-23b5-4ef8-9570-a942008409a5
3b4d3fb7-d950-4e9d-b079-a942008409a5
1166afe4-e0bd-4cd3-b4f1-a942008409a5
e1ea5d23-8e6d-4bab-a06f-a942008409a5
ef7bfe90-ce6c-45f2-92ee-a942008409a5
fbe82418-930d-484d-a6ae-a942008409a5
037f7dc6-00fd-4514-b0ad-a942008409a5
e790dc82-49e1-41f5-b75f-a942008409a4
fc09ad54-ee21-4fe9-973f-a942008409a5
a6b1c6ba-60f8-46ea-b806-a942008409a4
f87a8891-58d7-4d88-a15e-a942008409a4
7496233f-97ec-42f3-a599-a942008409a5
215352eb-f74d-4c5a-aa5f-a942008409a5
357e7ec7-91f1-4db8-9b75-a942008409a5
1c6043b7-42ce-4a25-9708-a942008409a5
9a7f6786-b326-47a3-b569-a942008409a5
de0daddb-c002-4b04-ba56-a942008409a5
80d6cc4a-2231-4a59-8b1f-a942008409a5
a007c866-0d98-4996-bfe7-a942008409a5
72ff4287-9108-4072-81fd-a942008409a4
f6130bda-9fe1-4135-bcdf-a942008409a5
db9890a7-a4bb-4936-b5de-a942008409a5
42d9ae52-7fbf-4d4c-aa0d-a942008409a4
5815f5a6-9b06-4f88-b4ab-a942008409a5
998de16d-a5c4-4dfc-af31-a942008409a4
c53013d8-f699-4538-8978-a942008409a5
8528bbd9-c589-4171-8356-a942008409a5
7685481d-86d7-43d2-b38f-a942008409a5
afe27519-563e-47c7-a0c7-a942008409a5
befe90d5-30ca-459e-9dac-a942008409a5
df4dfcd7-48c1-4b1d-b543-a942008409a4
741ae853-c19d-4189-8119-a942008409a4
8b77b39d-3005-483e-b2e8-a942008409a5
f2641646-c12d-4d57-9e97-a942008409a5
182e7949-9fea-49fb-9b8a-a942008409a5
3e408f99-2143-4ce3-9a71-a942008409a4
819254c8-35a4-45d8-a9da-a942008409a5
b89a3130-b5d7-4b95-b0a9-a942008409a5
74e88442-aa45-4762-8da0-a942008409a5
89956967-45bc-4288-8286-a942008409a5
ed46284e-8c7d-4f36-9d5f-a942008409a5
6df44ae0-11fa-4695-873e-a942008409a5
aceae15c-4219-42af-ad16-a942008409a5
b77902ba-0837-49d2-8bea-a942008409a5
199568ea-6243-4f67-b9bd-a942008409a5
8433d059-9637-4f9b-bdb7-a96700c9deec
'
```
</details>

**Comment by blacs30** [comment]
Hi @rasmusbe,
thanks for reporting the issue and this PR. I am not sure if the change to the third item really solves the issue. I think the issue is at the beginning of the python scripts.

I think this might better reliably fix it:
```
- details = sys.argv[1]
- details = details[1:-1].replace('"', '').split()
+ details = sys.argv[1].replace('"','')
+ details = details.splitlines()
+ details.pop(0)

```

Line 431,1226,1310

Here is also a fixed version. Would be great if you could test this as well and confirm that the issue is fixed.

Thanks.
[Bitwarden CLI.alfredworkflow.zip](https://github.com/blacs30/bitwarden-alfred-workflow/files/2424526/Bitwarden.CLI.alfredworkflow.zip)

**Comment by rasmusbe** [comment]
Ah, I see! Your fix works fine! But I found another problem, python print-function adds a new line at the end as default.

So I recommend you use
```python
sys.stdout.write(output.strip())
sys.stdout.flush()
```
instead of `print(output.strip())`

**Comment by blacs30** [comment]
Thanks for the recommendation.
I've implemented it in [version 1.2.3](https://github.com/blacs30/bitwarden-alfred-workflow/releases/tag/1.2.3), including the original fix for this issue.
